unified-installer/astra-unified-installer.sh

```diff
@@ -274,27 +274,27 @@ Optional Environment Variables:
-  TRIDENT_OPERATOR_IMAGE_REGISTRY    The Docker registry to pull the Trident Operator image from.
-  TRIDENT_AUTOSUPPORT_IMAGE_REGISTRY The Docker registry to pull the Trident Autosupport image from.
-  TRIDENT_IMAGE_REGISTRY             The Docker registry to pull the Trident image from.
+  TRIDENT_OPERATOR_IMAGE_REGISTRY    The Docker registry to pull the Astra Trident Operator image from.
+  TRIDENT_AUTOSUPPORT_IMAGE_REGISTRY The Docker registry to pull the Astra Trident Autosupport image from.
+  TRIDENT_IMAGE_REGISTRY             The Docker registry to pull the Astra Trident image from.
   CONNECTOR_OPERATOR_IMAGE_REGISTRY  The Docker registry to pull the Connector Operator image from.
   CONNECTOR_IMAGE_REGISTRY           The Docker registry to pull the Connector image from.
   NEPTUNE_IMAGE_REGISTRY             The Docker registry to pull the Neptune image from.
-  TRIDENT_ACP_IMAGE_REGISTRY         The Docker registry to pull the Trident ACP image from.
+  TRIDENT_ACP_IMAGE_REGISTRY         The Docker registry to pull the Astra Trident ACP image from.
 
   IMAGE_BASE_REPO                    The default base repository path (i.e. excluding the image name) for all images. Overridden by values below.
-  TRIDENT_OPERATOR_IMAGE_REPO        The repository path for the Trident Operator image.
-  TRIDENT_AUTOSUPPORT_IMAGE_REPO     The repository path for the Trident Autosupport image.
-  TRIDENT_IMAGE_REPO                 The repository path for the Trident image.
+  TRIDENT_OPERATOR_IMAGE_REPO        The repository path for the Astra Trident Operator image.
+  TRIDENT_AUTOSUPPORT_IMAGE_REPO     The repository path for the Astra Trident Autosupport image.
+  TRIDENT_IMAGE_REPO                 The repository path for the Astra Trident image.
   CONNECTOR_OPERATOR_IMAGE_REPO      The repository path for the Connector Operator image.
   CONNECTOR_IMAGE_REPO               The repository path for the Connector image.
   NEPTUNE_IMAGE_REPO                 The repository path for the Neptune image.
-  TRIDENT_ACP_IMAGE_REPO             The repository path for the Trident ACP image.
+  TRIDENT_ACP_IMAGE_REPO             The repository path for the Astra Trident ACP image.
 
-  TRIDENT_IMAGE_TAG                  The tag for the Trident image. Overridden by other 'TRIDENT_' values below.
-  TRIDENT_OPERATOR_IMAGE_TAG         The tag for the Trident Operator image.
-  TRIDENT_AUTOSUPPORT_IMAGE_TAG      The tag for the Trident Autosupport image.
-  TRIDENT_ACP_IMAGE_TAG              The tag for the Trident ACP image.
+  TRIDENT_IMAGE_TAG                  The tag for the Astra Trident image. Overridden by other 'TRIDENT_' values below.
+  TRIDENT_OPERATOR_IMAGE_TAG         The tag for the Astra Trident Operator image.
+  TRIDENT_AUTOSUPPORT_IMAGE_TAG      The tag for the Astra Trident Autosupport image.
+  TRIDENT_ACP_IMAGE_TAG              The tag for the Astra Trident ACP image.
```

Comment:
This is kind of redundant, since ACP == `Astra Control Provisioner`. So the whole thing is `Astra Trident Astra Control Provisioner`. Not sure what to do about it though 🤷‍♀️

Reply:
In the docs, we refer to Astra Trident and Astra Control Provisioner as two separate things.  ACP might be an extension of Trident, but we just say "Astra Control Provisioner".  
"Astra Trident ACP" should just be "Astra Control Provisioner".  From a PM standpoint, they are being treated as two separate things (even though they are not technically two separate products).  Thanks!